### DISCUSSION TITLE
After running the "udemy-enroller" in cli it shows the error

### DISCUSSION THREAD

**Discussion by vimalrajvs** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#298] [Q&A] [answered]
![udemy enroller error](https://user-images.githubusercontent.com/81948804/121480952-4230f600-c9e9-11eb-9125-f000b1a82a04.JPG)

**Comment by fakeid30** [comment, marked as answer]
This is not the proper way to run this script. Please re-read the README
